blingfire/src/lib.rs

```diff
@@ -1,100 +1,71 @@
 mod errors;
 
 use blingfire_sys::{TextToSentences as text_to_sentences_ffi, TextToWords as text_to_words_ffi};
-use failchain::{bail, ResultExt};
-use std::{convert::TryInto, mem};
+use failchain::ensure;
+use std::convert::TryInto;
+use std::i32;
+use std::os::raw::{c_char, c_int};
```

Comment:
nit. post-2018 I like having one `use` per library, e.g. `use std::{convert::TryInto, mem};`

Reply:
hmmm, that's not what rustfmt does and i think it's a less convenient to manage. but sure

Reply:
rustfmt doesn't have an opinion on this, cos it doesn't change either in the other one? why less convenient?